tmt/cli.py

```diff
@@ -482,6 +482,9 @@ def run_plans(context: Context, **kwargs: Any) -> None:
     help="""
          Filter by linked objects (regular expressions are supported for both relation and target).
          """)
+@option(
+    '--failed-only', is_flag=True, default=False,
+    help="Filter failed tests from a previous run to run again")
```

Comment:
I would say it's needed, or some other hint, because "filter failed tests from a previous run" does not say much about what will happen with them - filter them out, to run the passed ones only? Or filter them and run them and no other test? "... to run them again." was helping me understand the implication of "filtering".

Reply:
Well, this option is only about filtering. We can then extend what it is useful for, but for that it needs more options :)

```suggestion
    help="Filter failed tests from a previous run. Useful for rerunning failed tests.")
```

Reply:
I know it's just the filtering, but that's the point, the "to run again" was important for my understanding what the filtering will do - filter, yes, but will only the failed ones remain? Or the successful ones, and the failed ones would be filtered out of the set? "Filter failed tests" is not clear enough, it can fit both scenarios :)

Reply:
Rewrote it a bit. Hope it is better now.

Reply:
Sorry to rock the boat, but the current form doesn't work for me. "Filter failed tests from a previous run. Used when rerunning existing runs." - the option is very common, but the help is much less clear. How about "Ignore successful tests from the given run, discover only the failed ones. Use when you wish to re-run failed tests only from a given run. Requires either `--id` or `--last` option."

Reply:
@KwisatzHaderach let's go with @happz suggestion.

I am personally fine already ...